src/ntlm_crypto.c

```diff
@@ -577,6 +577,12 @@ int ntlm_reset_rc4_state(uint32_t flags, bool recv,
     return ret;
 }
 
+void ntlm_release_rc4_state(struct ntlm_signseal_state *state)
```

Comment:
Can you use this function in gssntlm_delete_sec_context() given you created it ?

Reply:
yep, good idea...